codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/ServerHttpBoundProtocolGenerator.kt

```diff
@@ -207,9 +209,31 @@ private class ServerHttpBoundProtocolTraitImplGenerator(
                         )
                 }
             }
+
```

Comment:
nit: here and in line 223, these are empty lines, when we want them to be indented 12 spaces, so that the indent of the until the start of the block is constant.